apps/morph_check

```diff
@@ -84,13 +84,15 @@ Default errors checked for
 * Zero radius points
 * Zero length segments
 * Zero length sections
+* Jumps in the Z-direction
 
 Default values for options
 --------------
 * has_nonzero_soma_radius 0.0
 * has_all_nonzero_neurite_radii: 0.007
 * has_all_nonzero_segment_lengths: 0.01
 * has_all_nonzero_section_lengths: 0.01
+* has_no_zjumps: 30 micrometer - distance considered a jump
```

Comment:
Do we really want this switched on by default? Especially given that it arbitrarily picks one axis, so is not very general. We could let users enable this in their config file.